coldfront/core/project/views.py

```diff
@@ -2403,32 +2403,42 @@ def show_pooled_project_selection_form_condition(wizard):
 class SavioProjectRequestListView(LoginRequiredMixin, TemplateView):
     template_name = 'project/project_request/savio/project_request_list.html'
     login_url = '/'
-
     # Show completed requests if True; else, show pending requests.
     completed = False
 
+    def get_queryset(self):
+        order_by = self.request.GET.get('order_by')
+        if order_by:
+            direction = self.request.GET.get('direction')
+            if direction == 'asc':
+                direction = ''
+            else:
+                direction = '-'
+            order_by = direction + order_by
+        else:
+            order_by = 'id'
+
+        return SavioProjectAllocationRequest.objects.order_by(order_by)            
+
     def get_context_data(self, **kwargs):
         """Include either pending or completed requests. If the user is
         a superuser, show all such requests. Otherwise, show only those
         for which the user is a requester or PI."""
-        context = super().get_context_data(**kwargs)
 
         args, kwargs = [], {}
-
-        user = self.request.user
-        if not user.is_superuser:
-            args.append(Q(requester=user) | Q(pi=user))
-
+        context = super().get_context_data(**kwargs)
         if self.completed:
             status__name__in = ['Approved - Complete', 'Denied']
+            kwargs['status__name__in'] = status__name__in
```

Comment:
Both the `if` and `else` blocks have these same three lines, so we can refactor them out.